### DISCUSSION TITLE
2008/06/04/%E5%8F%88%E5%A4%B1%E7%9C%A0%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#223] [Announcements]
# 2008/06/04/%E5%8F%88%E5%A4%B1%E7%9C%A0%E4%BA%86

今天早上头疼的很，因为凌晨三点的时候突然醒过来就再睡不着了。据我爸说，我这是从他那遗传来的毛病。睡不着觉的时候就会瞎琢磨。今早躺在床上想来想去，就想到了鲁迅。

https://blog.qizhen.xyz/2008/06/04/%E5%8F%88%E5%A4%B1%E7%9C%A0%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

刘伟印
2008年08月25日 17:26 编辑
“以向鲁迅以及那个允许他发表文章的时代表示敬意”
顶你这句话，呵呵

回复
晶
2008年07月15日 14:41 编辑
你不是我们公司的吧？。。。

回复
Moon
2008年06月23日 11:12 编辑
其他周老先生的文才并不算高,但眼光毒,笔锋利,所以~
 
你看过李敖的书没,张扬的过了头,但不可否认的是他的言语比鲁迅更犀利,杀人与言语间,骂人于无形中
 
我也失眠,可能太喜欢熬夜了,以致白天像绵羊,晚上化身为猫头鹰

回复
竹贤
2008年06月7日 09:56 编辑
如若失眠，莫不如起来做一些自己平时喜欢做的事情。坐在床上看看书也可，但应该读一些轻松的、休闲类的书籍。也可以听听轻音乐，如催眠曲一类轻松抒情的乐曲，让自己慢慢陶醉在其中。
用脑过度，运动不足，大脑高度兴奋，容易失眠。每天适度运动。
还有一个建议，把吉他重新捡起来吧，每晚消遣半小时，也许对你的失眠会有疗效。

回复
WL
2008年06月6日 21:17 编辑
COOL

回复
Srain
2008年06月5日 11:20 编辑
我也喜欢鲁迅，嗯

回复
